.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_DOMAIN="https://webbtc.dev"
```

Comment:
@MiguelMedeiros Is that a viable way to get absolute paths? Does this work in combination with vercel?

Reply:
Yes, I can set this env variable on vercel.